pkg/azuredisk/controllerserver_test.go

```diff
@@ -1481,13 +1481,7 @@ func TestGetCapacity(t *testing.T) {
 }
 
 func TestListVolumes(t *testing.T) {
-	d, _ := NewFakeDriver(t)
```

Comment:
pls enable ut after this PR(https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/627) merged.

Reply:
https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/627 is merged